lib/cyclopedio/syntax/name_decorator.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'stanford/all'
+require 'wiktionary/noun'
+
+module Cyclopedio
+  module Syntax
+
+    class NameDecorator < SimpleDelegator
```

Comment:
SimpleDelegator is rather unefficient since (as far as I remember) it uses method_missing.